src/tests/functional/plugin/shared/src/behavior/ov_plugin/properties_tests.cpp

```diff
@@ -81,8 +81,10 @@ void OVPropertiesTestsWithCompileModelProps::SetUp() {
     std::tie(temp_device, properties) = this->GetParam();
 
     std::string::size_type pos = temp_device.find(":", 0);
-    std::string hw_device;
-    if (pos == std::string::npos) {
+    std::string hw_device = "";
+    if (temp_device == "AUTO") {
+        target_device = temp_device;
+    } else if (pos == std::string::npos) {
```

Comment:
1. If Find : in temp_device
2. Find target_device and set device::priorites
3. Else
4. target_device = temp_device